samples/storage_csm_powerscale_v270.yaml

```diff
@@ -287,7 +287,14 @@ spec:
           # Default value: powerstore
           - name: "X_CSI_REPLICATION_CONTEXT_PREFIX"
             value: "powerscale"
-      
+
+      - name: metadataretriever
+        # image: Image to use for csi-metadata-retriever. This shouldn't be changed
+        image: dellemc/csi-metadata-retriever:v1.3.0
+        envs:
+          - name: "CSI_RETRIEVER_ENDPOINT"
+            value: "/var/run/csi/csi_retriever.sock"
```

Comment:
Can this value be changed? if yes, is that tested. I dont see the corresponding code for that

Reply:
retriever endpoint value is fixed